### PR TITLE
feature/CLS2-273-use-inline-svg-for-toggle

### PR DESCRIPTION
**Pull request by chopkinsmade**
## Description of change

Replace the imported caret.svg with an inline svg. This allows the caret to have it's colour changed using a prop instead of always being black

## Test instructions

No visible changes in this PR

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/components/ToggleSection/BaseToggleSection.jsx [hunk 1]
```diff
@@ -38,15 +36,31 @@ export const ToggleButton = styled('button')`
...
 
+export const ToggleButtonIcon = ({ colour = BLACK, isOpen }) => (
+  <ToggleButtonIconStyle isOpen={isOpen}>
+    <svg
+      width="18px"
+      height="18px"
+      version="1.1"
+      viewBox="210 275 600 425"
+      xmlns="http://www.w3.org/2000/svg"
+      class="app-c-expander__icon app-c-expander__icon--up"
```
Comment (paulgain) [inline src/client/components/ToggleSection/BaseToggleSection.jsx:54]: Where are these classes coming from?

**Review by paulgain — APPROVED** [review]

File: src/client/components/ToggleSection/BaseToggleSection.jsx (hunk 1 above)
Comment (chopkinsmade) [reply src/client/components/ToggleSection/BaseToggleSection.jsx:54]: They were part of the `caret.svg` file that got replaced, along with the rest of those svg attributes.

Comment (paulgain) [reply src/client/components/ToggleSection/BaseToggleSection.jsx:54]: Yes, but are they actually needed? A search of `expander` within the codebase yields no results.

**Review by PaulWheatcroft — APPROVED** [review]

File: src/client/components/ToggleSection/BaseToggleSection.jsx (hunk 1 above)
Comment (chopkinsmade) [reply src/client/components/ToggleSection/BaseToggleSection.jsx:54]: Removed on local and doesn't seem to have an impact, will get rid of them

Comment (paulgain) [reply src/client/components/ToggleSection/BaseToggleSection.jsx]: Thanks.

**Pull request merged**